python/mlc_chat/compiler/model/llama_weight.py

```diff
@@ -59,3 +61,18 @@ def huggingface(model_config: LlamaConfig, _) -> ExternMapping:
             map_func[name] = lambda x: x
             param_map[name] = [name]
     return ExternMapping(param_map, map_func, unused_params)
+
+
+def group_quant(
```

Comment:
I don't have strong opinion on where to place the `group_quant` function, just to confirm are we getting rid of all `{model_name}_parameter.py` and `{model_name}_quantization.py` and consolidate to `{model_name}_weight.py`?

Reply:
good point! I think `llama_loader.py` and `llama_quantization.py` should make most of the sense